### ISSUE TITLE
Disc Numbers not working.

### ISSUE THREAD

**Issue by Fireokiller** [un-pogaz/Now-Playing-in-Texts#9]
This is a great plug-in for VLC!  I use it to pull the metadata from each video in VLC and display it on our 24/7 Stream.

I cannot seem to get the Disc Number to show up in any of the TEXT output files. 
I have copied and changed the code a few times changing the "{DISCNUMBER}" to  "{discnumber}" and also tried "{disc_number}" after looking up some of the VLC 3.0 LUA code on the DEV site. 

In all of our video files Disc Numbers are entered in using MP3TAG and range from one digit # to three ### max.
The Disc Number shows up without issue in the VLC Playlist, and I have NO issues with the Track Numbers showing up in the NP TEXT Output files.  

With my learning disabilities and no skills in coding I need some help if someone gets a few minutes.
Thank You! 

PS: I have noticed misspelled code words like Lenght instead of Length and Valide instead of Valid (I don't think this causes the issue.)

**Comment by un-pogaz** [comment]
Sometime, the metadata reading is a bit capricious depending on the file format, because all formats don't support all metadata. It seems that in this case, the video/MP4 don't fully support "{discnumber}", and so will be always output a empty value.

In case of doubt, there are the files `np_metadata.txt` and `np_metadata_full.txt` which respectively list all usable values for this file and all usable values in "Now Playing in Texts", even the empty ones. If the value you're looking for isn't here, then you can't use it.

**"But VLC read it the discnumber well?"**

Yes, and I have no idea how. On my side, this value appear nowhere and I realy try hard many different ways. My theory is that the VLC's LUA module is incomplete/imperfect/buged and so the "discnumber" is never transfered to my code.

tldr: "{discnumber}" is not supported for video/MP4 file, not my fault.

**Comment by Fireokiller** [comment]
Ok Thank you. 

I just loaded MP3 files in VLC and ALL of them do not show a DISC NUMBER in the VLC PLAYLIST BIN, but ALL of the video files .mp4s do show a DISC NUMBER up to three digits. 

Yes It's got to be the VLC code not passing this information off into your LUA script. 
Hopefully someone might be able to find the issue in the VLC code and it can be fixed one day. 

Thank you for taking a look for me, and thanks for making the plug-in!

-John
